src/standalone/Soil/energy_hydrology.jl

```diff
@@ -158,6 +166,13 @@ function EnergyHydrology{FT}(;
     sources::Tuple,
     lateral_flow::Bool = true,
 ) where {FT, D, PSE}
+    top_bc = boundary_conditions.top
+    if typeof(top_bc) <: AtmosDrivenFluxBC
+        # If the top BC indicates atmospheric conditions are driving the model
+        # add baseflow as a sink term
+        subsurface_source = subsurface_runoff_source(top_bc.runoff)
+        sources = append_source(subsurface_source, sources)
```

Comment:
we will use the same functionality for sublimation, when we add it, since that is a source term that only makes sense for this BC.